### DISCUSSION TITLE
Welcome to CALDERA Discussions!

### DISCUSSION THREAD

**Discussion by wbooth** [mitre/caldera#1969] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about how you are using CALDERA.

**Comment by digoblin** [comment]
Hey there and thank you very much for this wholesome project, it seems to have great potential!

I'm working as part of a SOC team in Portugal and currently I'm trying to implement Caldera to improve the visibility of our tools in detecting common attack patterns/techniques.

Already knew about Caldera for some time, but only now got the chance to use it.

After testing it, it definitely seems capable to improve our work, and since it uses Python I feel more confident that I can extend or change it to fit our needs.
